backend/app/Models/ReleaseNote.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Models;
+
+use App\Enums\ReleaseNoteGenre;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class ReleaseNote extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        "genre", "title", "description", "date"
+    ];
+    /**
+     * 日付の登録(format使えるために)
+     *
+     * @var array
+     */
+    protected $dates = ['date'];
+    /**
+     * Enumキャスト
+     *
+     * @var array
+     */
+    protected $casts = [
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>PHPDoc type array of property App\Models\ReleaseNote::$casts is not covariant with PHPDoc type array<string, string> of overridden property Illuminate\Database\Eloquent\Model::$casts.